examples/rust/profile/src/main.rs

```diff
@@ -224,6 +239,9 @@ struct Args {
     /// Increase verbosity (can be supplied multiple times).
     #[arg(short = 'v', long = "verbose", global = true, action = ArgAction::Count)]
     verbosity: u8,
+    /// Software event
```

Comment:
May be worth it to elaborate a bit more. Perhaps something like:
```suggestion
    /// Use software event for triggering stack trace capture.
    ///
    /// This can be useful for compatibility reasons if the hardware event is not available
    /// (which could happen in a virtual machine, for example).
```

Reply:
Completed the help info. Also add it in c version.

Reply:
In c, the help info now:
```text
Usage: ./profile [-f <frequency>] [--sw-event] [-h]
Options:
  -f <frequency>  Sampling frequency [default: 1]
  --sw-event      Use software event for triggering stack trace capture
  -h              Print help
```